### PR TITLE
Some Description Fixes and Oversights

### PR DESCRIPTION
**Pull request by EmilitiaEnnehrt**
## About The Pull Request

Small batch fixes:
* Fixes grammar and spelling of a few things that was reported. Also fix an inaccuracy
* Makes it so that ammunition is less confusing in the description.
* Makes it so that the Prime helmet reflects their sprite by showing ears, eyes, etc for those that has open top/shown face.
* Makes it so that the halberd cannon actually counts as a blade and can accept mods related to it.
* Deconstructing crates and closets now requires user to be on any intent but help.

## Changelog
:cl:
fix: some descriptions
fix: Paladin crown not showing ears
tweak: deconstructing storage qol
/:cl:

### CODE REVIEW

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -420,7 +420,7 @@
 	name = "sink"
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "sink"
-	desc = "A HV-11 sink. Used for washing the hands and face."
+	desc = "A HV-11 sink used for washing hands and face."
```
Comment (Takykardi) [inline code/game/objects/structures/watercloset.dm:423]: Suggest also changing the first "A" to "An HV-11..." due to the quirks of when to use A vs. An and the phonetics of saying the letters 'HV'.

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 2]
```diff
@@ -32,7 +32,7 @@
 /obj/item/rig/industrial
 	name = "industrial suit control module"
 	suit_type = "industrial hardsuit" //Credit for sprites go to Près de l'oiseau over at Eris
-	desc = "A heavy, powerful rig used by construction crews and mining corporations. The soft plating can protect the wearer from unexpected hazards, but is not reliable in prolonged combat."
+	desc = "A heavy, powerful rig used by construction crew and mining corporations. The soft plating can protect the wearer from unexpected hazards, but is not reliable in prolonged combat."
```
Comment (Takykardi) [inline code/modules/clothing/spacesuits/rig/suits/station.dm:35]: Please revert. Corporations is plural and crews should remain plural.

File: code/modules/projectiles/ammunition/bullets.dm [hunk 3]
```diff
@@ -311,7 +311,7 @@
 	projectile_type = /obj/item/projectile/bullet/heavy_rifle_408/practice
 
 /obj/item/ammo_casing/heavy_rifle_408/hv
-	desc = "A 8.6mm high-velocity bullet casing."
+	desc = "A 8.6mm high-velocity SLAP bullet casing."
```
Comment (Takykardi) [inline code/modules/projectiles/ammunition/bullets.dm:314]:
```suggestion
	desc = "An 8.6mm high-velocity SLAP bullet casing."
```
English is weird. Due to the word eight, should be 'An'. Good opportunity to make sure other 8.6mm casings have the same grammar.

File: code/modules/projectiles/ammunition/bullets.dm [hunk 4]
```diff
@@ -495,7 +495,7 @@
 
 /obj/item/ammo_casing/antim/lethal
 	name = "\"Red-Nose\" shell casing"
-	desc = "A SA 14.5mm Anti-Materiel \"Red-Nose\" shell."
+	desc = "A SA 14.5mm Anti-Materiel \"Red-Nose\" shell, punches organics harder than the standard round."
```
Comment (Takykardi) [inline code/modules/projectiles/ammunition/bullets.dm:498]:
```suggestion
	desc = "An SA 14.5mm Anti-Materiel \"Red-Nose\" shell, punches organics harder than the standard round."
```
Same quirk as above. "An 'ess-ayy'..." as pronounced. Also applies to all other SA 14.5mm ammo.

**Review by Takykardi — CHANGES_REQUESTED** [review]
Minor grammar things. The English language is nothing but pain and suffering. Other changes all look good though!

File: code/game/objects/structures/watercloset.dm (hunk 1 above)
Comment (EmilitiaEnnehrt) [reply code/game/objects/structures/watercloset.dm:423]: This is fair, yes!

File: code/modules/clothing/spacesuits/rig/suits/station.dm (hunk 2 above)
Comment (EmilitiaEnnehrt) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:35]: This is not grammatically correct. Crew is already a nebulous term that is both singular and plural. "Crew" is the plural of "crew".

In American and British English standard, unless I've missed something, the word "crews" is an unrecognized slang.

File: code/modules/psionics/psionic_items/psi_armor.dm
[the remark this answers is not in the document]
Comment (EmilitiaEnnehrt) [reply code/modules/psionics/psionic_items/psi_armor.dm:1]: Trilby says its autoconversion.

File: code/modules/clothing/spacesuits/rig/suits/station.dm (hunk 2 above)
Comment (Takykardi) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:35]: Merriam-Webster suggests crews is a correct form, I'd have to dig deeper. This language sucks.

**Review by Takykardi — APPROVED** [review]
New changes look good. Ready to go, unless the dictionary-diving turns up something of crew vs crews. Either way, everything checks out.

File: code/modules/clothing/spacesuits/rig/suits/station.dm (hunk 2 above)
Comment (Shadow-Quill) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:35]: Crews is correct, as it implies the rig is used by multiple groups of construction workers.

Comment (EmilitiaEnnehrt) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:35]: All I'm commenting is that my contempt for American English grows, I'll adamantly disagree on this in spirit. Changes made.

File: code/datums/topic/admin.dm [hunk 5]
```diff
@@ -1051,14 +1051,14 @@
...
-		to_chat(source.owner, "Standby!  Reload cycle in progress!  Gunnary crews ready in five seconds!")
+		to_chat(source.owner, "Standby! Reload cycle in progress! Gunnery crew ready in five seconds!")
 		return
 
 	BSACooldown = TRUE
 	spawn(50)
 		BSACooldown = FALSE
 
-	to_chat(M, "You've been hit by bluespace artillery!")
+	to_chat(M, "You've been hit by Bluespace Artillery!")
```
Comment (lolman360) [inline code/datums/topic/admin.dm:1061]: neither bluespace nor artillery are proper nouns, the original non-capitalization was fine

```suggestion
	to_chat(M, "You've been hit by bluespace artillery!")
```

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 6]
```diff
@@ -126,7 +126,7 @@ Advanced Voidsuit: Guild Master
 /obj/item/rig/ce
 	name = "advanced hardsuit control module"
 	suit_type = "advanced hardsuit"
-	desc = "An advanced hardsuit that protects against hazardous, low pressure environments. Shines with a high polish."
+	desc = "An advanced hardsuit that protects against hazardous, low pressure environments. Shines with high polish."
```
Comment (lolman360) [inline code/modules/clothing/spacesuits/rig/suits/station.dm:129]: 'a high polish' is correct here.
```suggestion
	desc = "An advanced hardsuit that protects against hazardous, low pressure environments. Shines with a high polish."
```

**Review by lolman360 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**